tensorflow/core/kernels/cwise_op_equal_to_1.cc

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 namespace tensorflow {
 REGISTER7(BinaryOp, CPU, "Equal", functor::equal_to, float, Eigen::half, double,
           uint8, int8, int16, bfloat16);
+REGISTER3(BinaryOp, CPU, "Equal", functor::equal_to, uint16, uint32, uint64);
```

Comment:
If we have a `REGISTER10`, we should merge the two calls

Reply:
Thanks @mihaimaruseac. I checked `tensorflow/core/kernels/cwise_ops_common.h` it is only up to `REGISTER9`.